helm/templates/deployment.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.app.name }}
+  labels:
+    app: {{ .Values.app.name }}
+    group: {{ .Values.app.group }}
+spec:
+  replicas: {{ .Values.app.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Values.app.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.app.name }}
+        group: {{ .Values.app.group }}
+    spec:
+      containers:
+        - name: {{ .Values.app.name }}
+          image: {{ .Values.app.container.image }}
+          ports:
+            - containerPort: {{ .Values.app.container.port }}
+          env:
+            {{- range $key := .Values.app.container.env }}
+            - name: {{ $key }}
+              valueFrom:
+                secretKeyRef:
+                  name: users-service-staging
```

not sure why i can't do this
```suggestion
                  name: {{ .Values.app.name }}-{{ .Values.app.environment }}
```